src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -753,3 +848,37 @@ CATCH_RETURN()
 
     return _userLocaleName;
 }
+
+// Routine Description:
+// - Digs through the directory that the current executable is running within to find
+//   any TTF files sitting next to it.
+// Arguments:
+// - <none>
+// Return Value:
+// - Iterable collection of filesystem paths, one per font file that was found
+[[nodiscard]] std::vector<std::filesystem::path> DxFontRenderData::s_GetNearbyFonts()
+{
+    std::vector<std::filesystem::path> paths;
+
+    // Find the directory we're running from then enumerate all the TTF files
+    // sitting next to us.
+    const std::filesystem::path module{ wil::GetModuleFileNameW<std::wstring>(nullptr) };
+    const auto folder = module.parent_path();
+
+    for (auto& p : std::filesystem::directory_iterator(folder))
+    {
+        if (p.is_regular_file())
+        {
+            auto extension = p.path().extension().wstring();
+            std::transform(extension.begin(), extension.end(), extension.begin(), std::towlower);
+
+            const std::wstring_view ttfExtension{ L".ttf" };

```

Comment:
```suggestion
            static constexpr std::wstring_view ttfExtension{ L".ttf" };
```

Reply:
otherwise it gets initialized as from pointer+length pair every iteration of the loop (worst case)